### ISSUE TITLE
Github php fatal error

### ISSUE THREAD

**Issue by makitso** [fisharebest/webtrees#3200]
I am seeing these errors in the error report.
**Error:**
[Thu Apr 30 20:22:50.405993 2020] [fcgid:warn] [pid 14266:tid 140293035276032] [host skatekey.net] [client 2400:8901::f03c:91ff:fee9:70fe:43208] PHP Fatal error: Uncaught ErrorException: Trying to get property 'name' of non-object in /var/www/html/sk/skatekey.net/resources/views/layouts/default.phtml:68

**Access Log:**
2400:8901::f03c:91ff:fee9:70fe - - [30/Apr/2020:20:22:49 -0700] "HEAD / HTTP/1.1" 301 221 "-" "Wget/1.18 (linux-gnu)"
2400:8901::f03c:91ff:fee9:70fe - - [30/Apr/2020:20:22:49 -0700] "HEAD / HTTP/1.1" 500 6161 "-" "Wget/1.18 (linux-gnu)"
2400:8901::f03c:91ff:fee9:70fe - - [30/Apr/2020:20:22:52 -0700] "GET / HTTP/1.1" 302 6261 "-" "Wget/1.18 (linux-gnu)"

**Comment by fisharebest** [comment]
The error means that you are trying to display a "webtrees page" (i.e. headers and footers) without having a defined theme.

This most likely means that there was an earlier error (which was caught), and then a second error occured during the error handler.

The earlier error must have been very early in the bootstrap process (before we initialise the theme).

I note that this IP addresses belongs to linode (a server farm), and is making a command-line or scripted call to fetch your site (using `wget`).  This is not a regular user, nor is it a search spider.  

FYI, linode is in the "provider blacklist" in the "BadBotBlocker" middleware.
In other words, webtrees has code to explicitly block requests from linode servers.
This code is commented out by default.

**Comment by makitso** [comment]
Thank you Greg.
